src/store/Reducer.ts

```diff
@@ -4,10 +4,16 @@ import Cookies from "universal-cookie";
 
 import { State } from "./State";
 import * as Actions from "./Actions";
-import { admin } from "./Textile";
+import { admin, Context } from "./Textile";
 
 export const cookies = new Cookies();
 
+const maxAge = 60 * 60 * 24 * 7; // TODO: Pick a better time (1 week)
+
+const pickupCookies = (context: Context) => {
```

Comment:
Helpful function to pick up any cookies we need along the way. We can just call this any time a remote call goes out. That way, if we've changed orgs or something between calls, we can update the headers before the API call goes out.